consoles/sshVirtsh.pm

```diff
@@ -496,6 +498,22 @@ sub resume ($self) {
 
 sub get_remote_vmm ($self) { $bmwqemu::vars{VMWARE_REMOTE_VMM} // '' }
 
+sub _encode_config ($self, $config, $key) {
+    # expand path
+    $config = sprintf('%s/data/%s', $bmwqemu::vars{CASEDIR}, $config);
+
+    unless (-e -f $config) {
+        croak "'$config' either doesn't exist or it is not a file, update the $key variable";
+    }
```

Comment:
I think `-e` is redundant. To fool the coverage analysis (so you don't have to write a test case for this `croak`) you can use a post-unless:
```suggestion
    croak "'$config' either doesn't exist or it is not a file, update the $key variable" unless -f $config;
```

Reply:
fixed